src/components/pages/parameter-or-variable.jsx

```diff
@@ -34,7 +38,7 @@ const ParameterOrVariablePage = React.createClass({
     }
     return (
       <div>
-        <Link className="btn btn-default" to="/">Retour à l'accueil</Link>
+        <Link className="btn btn-default" onClick={this.handleBackButtonClick} to="/">Retour</Link>
```

Comment:
The main drawback I see with your proposition is that the user could not copy the link `href`, in an absolute manner (in the sense that going back is relative).